src/components/TextField/index.jsx

```diff
@@ -425,71 +420,79 @@ export default class TextField extends React.Component {
     } = this.state;
```

Comment:
line 316: add theme()?

Reply:
This is same scenario as the above comment. It would be possible to pass the theme to it here but also more complicated. I thought I would leave it for now until someone needs this themed.

Reply:
sounds good